### PR TITLE
feat: rename investor to dao-member

### PR DESCRIPTION
**Pull request by mohandast52**
### URL changed to https://member.autonolas.network/

*  Investor to Member
* buOlas description updated

ok

### CODE REVIEW

File: components/Home/BuOlas/index.jsx [hunk 1]
```diff
@@ -2,24 +2,14 @@ import { Typography } from 'antd/lib';
...
-      <a
-        href="https://github.com/valory-xyz/autonolas-governance/blob/main/docs/Specs%20of%20governance%20contracts_v1.1.0.pdf"
-        target="_blank"
-        rel="noreferrer"
-      >
-        <Text type="secondary" underline>
-          Learn more
-        </Text>
-      </a>
+      buOLAS is a token that is used for the project team members and
```
Comment (mariapiamo) [inline components/Home/BuOlas/index.jsx:11]:
```suggestion
buOLAS is a token vesting contract that allows to lock and linearly
      released OLAS tokens.
```

File: components/Home/BuOlas/index.jsx [hunk 2]
```diff
@@ -2,24 +2,14 @@ import { Typography } from 'antd/lib';
...
+      consultants.&nbsp;
```
Comment (mariapiamo) [inline components/Home/BuOlas/index.jsx:12]:
```suggestion
 .&nbsp;
```

Comment (mariapiamo) [reply components/Home/BuOlas/index.jsx:12]: The sentence that has to be removed is " Usually, this is used for the project team members
      and consultants"

File: HISTORY.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# Release History - `autonolas-investor-frontend`
+# Release History - `autonolas-dao-member-frontend`
```
Comment (mariapiamo) [inline HISTORY.md:1]:
```suggestion
# Release History - `autonolas-member-frontend`
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
-# Investor App Frontend
-This repository contains code for the [Investor App](https://investor.autonolas.network/) frontend.
+# Dao member App Frontend
+This repository contains code for the [Dao member App](https://dao-member.autonolas.network/) frontend.
```
Comment (mariapiamo) [inline README.md:2]:
```suggestion
This repository contains code for the [Member App](https://member.autonolas.network/) frontend.
```

File: SECURITY.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
 # Security Policy
 
-This document outlines security procedures and general policies for the `autonolas-investor-frontend` project.
+This document outlines security procedures and general policies for the `autonolas-dao-member-frontend` project.
```
Comment (mariapiamo) [inline SECURITY.md:3]:
```suggestion
This document outlines security procedures and general policies for the `autonolas-member-frontend` project.
```

File: SECURITY.md [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 
 ## Supported Versions
 
-The following table shows which versions of `autonolas-investor-frontend` are currently being supported with security updates.
+The following table shows which versions of `autonolas-dao-member-frontend` are currently being supported with security updates.
```
Comment (mariapiamo) [inline SECURITY.md:7]:
```suggestion
The following table shows which versions of `autonolas-member-frontend` are currently being supported with security updates.
```

File: SECURITY.md [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 
 | Version   | Supported          |
 | --------- | ------------------ |
 | `N/A`     | :x:                |
 
 ## Reporting a Vulnerability
 
-The `autonolas-investor-frontend` team and community take all security bugs in `autonolas-investor-frontend` seriously. Thank you for improving the security of `autonolas-investor-frontend`. We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
+The `autonolas-dao-member-frontend` team and community take all security bugs in `autonolas-dao-member-frontend` seriously. Thank you for improving the security of `autonolas-dao-member-frontend`. We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
```
Comment (mariapiamo) [inline SECURITY.md:15]:
```suggestion
The `autonolas-member-frontend` team and community take all security bugs in `autonolas-member-frontend` seriously. Thank you for improving the security of `autonolas-member-frontend`. We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
```

File: README.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
-# Investor App Frontend
-This repository contains code for the [Investor App](https://investor.autonolas.network/) frontend.
+# Dao member App Frontend
```
Comment (oaksprout) [inline README.md:1]: Please update to "DAO" rather than "Dao" everywhere

File: README.md (hunk 4 above)
Comment (oaksprout) [inline README.md:2]: Member, not member

**Review by mariapiamo — CHANGES_REQUESTED** [review]
Update dao-member with member

File: components/Home/BuOlas/index.jsx (hunk 1 above)
Comment (oaksprout) [reply components/Home/BuOlas/index.jsx:11]: Should be "buOLAS is a vesting contract that allows locking and linear release of OLAS tokens."

File: README.md (hunk 4 above)
Comment (oaksprout) [reply README.md:2]: everywhere

**Review by oaksprout — CHANGES_REQUESTED** [review]

**Review by mariapiamo — CHANGES_REQUESTED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**